cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -21,12 +21,21 @@
 from cg.meta.workflow.analysis import AnalysisAPI
 from cg.models.scout.scout_load_config import ScoutLoadConfig
 from cg.store import Store
-from cg.store.models import Analysis, Family, Sample
+from cg.store.models import Analysis, Customer, Family, Sample
 from housekeeper.store.models import File, Version
+from pydantic.dataclasses import dataclass
 
 LOG = logging.getLogger(__name__)
 
 
+@dataclass
+class RnaDnaBundle:
```

Comment:
Nice 🎆

Reply:
I am not fond of this being named Bundle for the reason that we have Bundles in Housekeeper - could you consider using of the the following suggestions?

- RNADNARelationship
- RNADNAConnection
- RNADNARecord

my personal favorite is the latest.

Also, always use RNA and DNA in all caps 😄

Reply:
Went with RNADNACollection!